src/DFC_BusinessOntology.owl

```diff
@@ -6774,6 +7289,11 @@
         <IRI>#DFC_BusinessOntology_Subject</IRI>
         <Literal xml:lang="fr">Sujet DFC</Literal>
     </AnnotationAssertion>
+    <AnnotationAssertion>
+        <AnnotationProperty abbreviatedIRI="rdfs:comment"/>
+        <IRI>#DefinedProduct</IRI>
+        <Literal xml:lang="en">Product : Any offered product or service. For example: a carrot, a box of fruits, or soap.</Literal>
```

Comment:
Here there is a dot at the end. We should align all the definitions on a same format.